contracts/football_bets.py

```
@@ -88,6 +110,15 @@ def create_bet(
     @gl.public.write
     def resolve_bet(self, bet_id: str) -> None:
+        """Resolve an existing bet for the caller using the configured result URL.
+
+        Args:
+            bet_id: Unique identifier of the bet to resolve.
+
+        Raises:
+            Exception: If the bet is already resolved or the match has not
+                finished yet.
+        """
         if self.bets[gl.message.sender_address][bet_id].has_resolved:
```

Ruff (0.15.6)

[error] `gl` may be undefined, or defined from star imports

(F405)